dask_ml/model_selection/_hyperband.py

```diff
@@ -12,6 +13,7 @@
 from ..datasets import make_classification
 from ..wrappers import Incremental
 from ._split import train_test_split
+from time import time
```

Comment:
If you want subsecond precision on windows then I recommend 

    from distributed.metrics import time